compose.yml

```diff
@@ -0,0 +1,118 @@
+services:
+  web:
+    build:
+      context: "web"
+      target: "production"
+    ports:
+      - "3000:3000"
+    healthcheck:
+      test: ["CMD-SHELL", "wget --spider http://localhost:3000 || exit 1"]
+    env_file: [".env"]
+    depends_on:
+      db-psql:
+        condition: "service_healthy"
+      temporal:
+        condition: "service_healthy"
+  db-psql:
```

Comment:
nit: Service naming consistency, please choose between `db-<service>` or `<service>-db`, same for temporal-db